airbrakes/hardware/imu.py

```diff
@@ -160,6 +160,10 @@ def _fetch_data_loop(self, port: str, frequency: int) -> None:
                                 setattr(imu_data_packet, f"{channel}X", matrix.as_floatAt(0, 1))
                                 setattr(imu_data_packet, f"{channel}Y", matrix.as_floatAt(0, 2))
                                 setattr(imu_data_packet, f"{channel}Z", matrix.as_floatAt(0, 3))
+                            case "estPressureAlt":
+                                # Converts the pressure altitude to meters and sets it to the data packet object
+                                # TODO: We can maybe change this in sensor connect
+                                setattr(imu_data_packet, channel, data_point.as_float() * FEET_TO_METERS)
```

Comment:
It is in meters already? E.g. see your graph of the interest launch in the readme, and then Donald's attached picture of the flight data from the altimeter in slack

Reply:
i'm tweaking I guess

Reply:
no my log file that I was using (idk which one lol) is in feet. I'll keep it as an option and set if false by default